### PR TITLE
make profile popover responsive

### PR DESCRIPTION
**Pull request by denisdimitrov**
Remove previously set height and add fancy design for the scrollbar

### CODE REVIEW

**Comment by denisdimitrov** [comment]
[Scrollability of the profile page not functional now we have badges in the operation](https://trello.com/c/cRrq8Ekj/588-scrollability-of-the-profile-page-not-functional-now-we-have-badges-in-the-operation)

File: src/components/organisms/ProfileModal/ProfilePopoverContent.scss [hunk 1]
```diff
@@ -2,7 +2,20 @@
 
 #profile-popover {
   &.popover {
-    height: unset;
+    &::-webkit-scrollbar-track {
```
Comment (IvanBuljovcic) [inline src/components/organisms/ProfileModal/ProfilePopoverContent.scss:5]: So, *BLIP* for _-moz-_ or _-ms-_ users, right?

File: src/components/organisms/ProfileModal/ProfilePopoverContent.scss [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
+      border-radius: 3px;
+      background-color: rgba(0, 0, 0, 0);
```
Comment (IvanBuljovcic) [inline src/components/organisms/ProfileModal/ProfilePopoverContent.scss:7]: Variable?

File: src/components/organisms/ProfileModal/ProfilePopoverContent.scss (hunk 1 above)
Comment (denisdimitrov) [reply src/components/organisms/ProfileModal/ProfilePopoverContent.scss:5]: When I first started we weren't supposed to support multiple browsers and the users should always use Chrome, maybe something changed. @thecadams Is the project running on Chrome only?

Comment (IvanBuljovcic) [reply src/components/organisms/ProfileModal/ProfilePopoverContent.scss:5]: I know its only Chrome (for now), its not a required change to include different browser prefixes, its just something to keep in mind long term

**Review by IvanBuljovcic — APPROVED** [review]

File: src/components/organisms/ProfileModal/ProfilePopoverContent.scss (hunk 1 above)
Comment (thecadams) [reply src/components/organisms/ProfileModal/ProfilePopoverContent.scss:5]: Chrome only; tolerable on Safari; this is fine to include.

**Review by thecadams — APPROVED** [review]
LGTM!

**Pull request merged**